### PR TITLE
Drop py2 and add `--min-py-version` flag

### PR DESCRIPTION
**Pull request by mxr**

### CODE REVIEW

File: setup_cfg_fmt.py [hunk 1]
```diff
@@ -192,14 +187,15 @@ def _python_requires(
         if (
                 env.startswith('py') and
                 len(env) == 4 and
-                env[2:].isdigit()
+                env[2:].isdigit() and
+                env[2] == '3'
```
Comment (asottile) [inline setup_cfg_fmt.py:191]: just noticed this (unrelated) -- I think this misses `py311` as an env name

File: setup_cfg_fmt.py [hunk 2]
```diff
@@ -292,15 +285,12 @@ def _py_classifiers(
...
-            minimum = (3, 0)
-        else:
-            minimum = (minimum[0], minimum[1] + 1)
+        minimum = (minimum[0], minimum[1] + 1)
 
     classifiers = [
         f'Programming Language :: Python :: {_v(v)}' for v in versions
     ]
-    if (3,) in versions and (2,) not in versions:
+    if (3,) in versions:
```
Comment (asottile) [inline setup_cfg_fmt.py:293]: is this ever false now?

Comment (mxr) [reply setup_cfg_fmt.py:293]: Only in this test: https://github.com/mxr/setup-cfg-fmt/blob/555ab889d6bbb1b950df32479e527ffde1038823/tests/setup_cfg_fmt_test.py#L1124-L1134

Should I remove it?

Comment (asottile) [reply setup_cfg_fmt.py:293]: oh interesting -- yeah let's just always set it 🤷

File: setup_cfg_fmt.py (hunk 1 above)
Comment (mxr) [reply setup_cfg_fmt.py:191]: #182

File: setup_cfg_fmt.py (hunk 2 above)
Comment (mxr) [reply setup_cfg_fmt.py]: Sure

File: setup_cfg_fmt.py [hunk 3]
```diff
@@ -194,12 +189,12 @@ def _python_requires(
     for env in _tox_envlist(setup_cfg):
         match = TOX_ENV.match(env)
         if match:
-            version = _to_ver('.'.join(match.groups()))
+            version = _to_ver(f'3.{match.group(0)}')
```
Comment (asottile) [inline setup_cfg_fmt.py:192]: you want `match[1]` I believe

Comment (mxr) [reply setup_cfg_fmt.py:192]: Ah right I changed this before running tests. TIL you can use `__getitem__` on matches..

Comment (asottile) [reply setup_cfg_fmt.py]: yep!  it's new in 3.6 :D

**Review by asottile — APPROVED** [review]
can probably remove the python 2 tests too if they're not valuable -- let me know what you think otherwise I'll just merge

**Comment by mxr** [comment]
Removed a test and updated some others

**Pull request merged**